test_schemas/en/test_list_collector.json

```diff
@@ -321,6 +321,43 @@
                                     ]
                                 }
                             }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "test-number-block",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "test-currency",
+                                        "description": "",
+                                        "label": "Currency Test",
+                                        "mandatory": false,
+                                        "type": "Currency",
+                                        "currency": "GBP"
+                                    },
+                                    {
+                                        "id": "test-square-kilometres",
+                                        "description": "",
+                                        "label": "Kilometres Square",
+                                        "mandatory": false,
+                                        "type": "Unit",
+                                        "unit_length": "short",
+                                        "unit": "area-square-kilometer"
+                                    },
+                                    {
+                                        "id": "test-decimal",
+                                        "description": "",
+                                        "label": "Decimal test to 2 decimal places",
+                                        "mandatory": false,
+                                        "type": "Number",
+                                        "decimal_places": 2
+                                    }
+                                ],
+                                "description": "",
+                                "id": "test-number-range-question",
+                                "title": "Please enter test values (non-mandatory)",
+                                "type": "General"
+                            }
```

Comment:
I don't think we should be adding a question to this schema to test this. Can we use the existing `test_list_summary_on_question.json` schema? That schema already has an additional non-list question in the section.

Reply:
Ah I missed this - I did initially look for an existing schema that fits the requirement of having a mid section Question block along with a ListCollector. Let me take a look